### PR TITLE
[Pending Review]Update guidelines-generate-eft-files.md

### PR DESCRIPTION
**Pull request by theley502**
# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

File: support/dynamics/gp/guidelines-generate-eft-files.md [hunk 1]
```diff
@@ -194,93 +194,85 @@ Q1: When does a support case for EFT assistance become a consulting expense? How
...
+Q8: What if my bank uses a file format that varies slightly from one of the standard formats offered in GP?
 
-Also enter a Product Suggestion in the [Ideas](https://experience.dynamics.com/ideas/) link so Microsoft can track it and consider this new functionality for a future enhancement.
+A8: The EFT file formats available in Microsoft Dynamics GP are set up to follow the US Government and NACHA (National Automated Clearing House Association) published formats. They're published standard requirements. The file formats currently offered in Microsoft Dynamics GP are coded to achieve the required **** fields in these standard formats. So if your bank requires a format that varies slightly from one of the standard formats offered in GP, or a more updated version of a format, you may need to pursue a consulting service for the customization. In the meantime, you could check with the bank to see if they would make an exception, or accept any other formats that Dynamics GP can do (such as one of the standard NACHA formats), or edit the EFT file in Notepad before sending it to the bank.
```
Comment (lucciz01) [inline support/dynamics/gp/guidelines-generate-eft-files.md:229]: Hi @theley502, can you help double check the "****" in line 229? Do we need to remove it? Thanks!

File: support/dynamics/gp/guidelines-generate-eft-files.md [hunk 2]
```diff
@@ -410,14 +374,14 @@ Field: EFT Next File Number
...
+A31: No, at this time, you aren't able to count/sum until after the lines are listed. So you wouldn't be able to count the addenda lines in the Detail line since they're printed after the Detail line, so you wouldn't be able to achieve the CTX format. Your options would include:
 
 1. Run 'one check per invoice' instead of 'one check per vendor', and hard code the addenda count =1 in the EFT file format.
-1. Ask the bank if they would make an exception or accept any other NACHA format such as the CCD or PPD format. (Many banks have accepted the 'detail + addenda' count in a footer line instead.)
-1. Seek out a third-party product that allows this functionality or 4) a dexterity customization.
+2. Ask the bank if they would make an exception or accept any other NACHA format such as the CCD or PPD format. (Many banks have accepted the 'detail + addenda' count in a footer line instead.)
+3. Seek out a third-party product that allows this functionality or 4) a dexterity customization.
```
Comment (lucciz01) [inline support/dynamics/gp/guidelines-generate-eft-files.md:387]: Hi @theley502, can you help double check the meaning of the "4)" in line 387? I'm not sure whether it's correct. Thanks!

File: support/dynamics/gp/guidelines-generate-eft-files.md [hunk 3]
```diff
@@ -293,68 +285,54 @@ A17: Pad Blocking or 'block counts' requires EFT files to have some lines in it
...
 
-Q26: The bank won't accept my EFT file because the sequence number in the EFT file name is -0001 and we already uploaded an EFT file with that same sequence number today. The next file we generated was also generated with the same -0001 sequence number.
+Q24: The bank won't accept my EFT file because the sequence number in the EFT file name is -0001 and we already uploaded an EFT file with that same sequence number today. The next file we generated was also generated with the same -0001 sequence number.
 
-Q26: The naming convention for the EFT files (on Microsoft Dynamics GP 10.0 and higher versions for EFT Payables, and Microsoft Dynamics GP 2010 and higher for EFT Receivables) is comprised as follows:
+Q24: The naming convention for the EFT files (on Microsoft Dynamics GP 10.0 and higher versions for EFT Payables, and Microsoft Dynamics GP 2010 and higher for EFT Receivables) is comprised as follows:
```
Comment (lucciz01) [inline support/dynamics/gp/guidelines-generate-eft-files.md:335]: Hi @theley502, I changed the "Q24" to "A24" for line 335.

File: support/dynamics/gp/guidelines-generate-eft-files.md (hunk 1 above)
Comment (theley502) [reply support/dynamics/gp/guidelines-generate-eft-files.md:229]: looks good

File: support/dynamics/gp/guidelines-generate-eft-files.md (hunk 2 above)
Comment (theley502) [reply support/dynamics/gp/guidelines-generate-eft-files.md:387]: looks OK

**Pull request merged**